webui/tests/cypress/integration/01-setup.spec.ts

```diff
@@ -3,11 +3,8 @@ import { DEFAULT_WAIT_TIME } from '../constants';
 describe('setup', () => {
   const recordSelector = 'tr.ant-table-row';
 
-  before(() => {
-    cy.login();
-  });
-
   beforeEach(() => {
+    cy.login();
     cy.visit('/det/experiments');
```

Comment:
**question:** is it worth extracting `/det` on the tests? We can leave it to a separate ticket when we migrate away from `/det` (like when we sunset the old api).

Reply:
it's a good idea but IMO there is already enough moving parts here I think we're best leaving that off for another pr.